src/entity/Settings.ts

```diff
@@ -72,4 +75,5 @@ export const SETTINGS_DEFAULT = {
   optOutCompanion: false,
   optOutWeeklyGoal: false,
   sortingEnabled: false,
+  campaignCtaPlacement: null,
```

Comment:
Ideally, we should have an enum to hold the values.
```suggestion
  campaignCtaPlacement: 'header',
```